### PR TITLE
Fix build issue

### LINKED ISSUE #25: Fail to build with go 1.14
**Describe the bug**
The project fails to build with go 1.14, outputting the following error:
```
driver/go/driver.go:28:2: build constraints exclude all Go files in /src/replicant/vendor/github.com/containous/yaegi/stdlib
make: *** [Makefile:8: build] Error 1
```

**To Reproduce**
The easiest and most efficient way is to change the Dockerfile version from 1.13 to 1.14. Afterwards run `docker build .`

**Expected behavior**
Normal build operation

**Desktop (please complete the following information):**
 - OS: Gentoo
 - GO version: `go1.14 linux/amd64`
 - GO env:
```
GO111MODULE=""
GOARCH="amd64"
GOBIN=""
GOCACHE="/home/jorge/.cache/go-build"
GOENV="/home/jorge/.config/go/env"
GOEXE=""
GOFLAGS=""
GOHOSTARCH="amd64"
GOHOSTOS="linux"
GOINSECURE=""
GONOPROXY=""
GONOSUMDB=""
GOOS="linux"
GOPATH="/home/jorge/go"
GOPRIVATE=""
GOPROXY="https://proxy.golang.org,direct"
GOROOT="/usr/lib/go"
GOSUMDB="sum.golang.org"
GOTMPDIR=""
GOTOOLDIR="/usr/lib/go/pkg/tool/linux_amd64"
GCCGO="/usr/bin/gccgo"
AR="ar"
CC="x86_64-pc-linux-gnu-gcc"
CXX="x86_64-pc-linux-gnu-g++"
CGO_ENABLED="1"
GOMOD="/home/jorge/github/replicant/go.mod"
CGO_CFLAGS="-g -O2"
CGO_CPPFLAGS=""
CGO_CXXFLAGS="-g -O2"
CGO_FFLAGS="-g -O2"
CGO_LDFLAGS="-g -O2"
PKG_CONFIG="pkg-config"
GOGCCFLAGS="-fPIC -m64 -pthread -fmessage-length=0 -fdebug-prefix-map=/tmp/go-build160057016=/tmp/go-build -gno-record-gcc-switches"
```

### PR DESCRIPTION
**Pull request by Gisson**
Closes #25

### CODE REVIEW

**Comment by Gisson** [comment]
@brunotm please give it a look.

File: go.mod [hunk 1]
```diff
@@ -6,7 +6,8 @@ require (
 	github.com/MontFerret/ferret v0.9.1-0.20191122143658-aa81df38d4a2
 	github.com/aws/aws-sdk-go v1.28.9
 	github.com/brunotm/log v0.3.2
-	github.com/containous/yaegi v0.6.2
+	github.com/brunotm/replicant v0.1.13
```
Comment (brunotm) [inline go.mod:9]: From where this dep comes from?

File: go.sum [hunk 2]
```diff
@@ -41,6 +45,8 @@ github.com/beorn7/perks v1.0.1 h1:VlbKKnNfV8bJzeqoa4cOKqO6bYr3WgKZxO8Z16+hsOM=
 github.com/beorn7/perks v1.0.1/go.mod h1:G2ZrVWU2WbWT9wwq4/hrbKbnv/1ERSJQ0ibhJ6rlkpw=
 github.com/brunotm/log v0.3.2 h1:LFK+yqlhgCdlEkXyRI1z2Pv/nbzkSoOzQNW9xvKMY5U=
 github.com/brunotm/log v0.3.2/go.mod h1:bVGKDCvyUtEyYMSplSR1ZA9rs0eoOKr0f2HH3Ki4oGg=
+github.com/brunotm/replicant v0.1.13 h1:HBptEQRjnTIBboftTif9wDZs6ilSbR6GVNbwDQ3f1tc=
+github.com/brunotm/replicant v0.1.13/go.mod h1:O3s3iPuZ81RkZ+UAM/RrIiZyEhe4g0IQQL2pRnwHpjI=
```
Comment (brunotm) [inline go.sum:49]: same as above..

**Review by brunotm — CHANGES_REQUESTED** [review]

**Comment by brunotm** [comment]
dup #29

**Pull request closed without merging**